moment-timezone.js

```diff
@@ -420,6 +451,32 @@
 		momentProperties._z = null;
 	}
 
+	/************************************
+		Guess Whitelist
+	************************************/
+
+	(function (whitelist) {
+		var zones, i, country;
+		for (country in whitelist) {
+			if (whitelist.hasOwnProperty(country)) {
+				zones = whitelist[country].split(' ');
+				for (i = 0; i < zones.length; i++) {
+					guessWhitelist.push(country + '_' + zones[i]);
+				}
+			}
+		}
+	}({
+		africa: 'lagos johannesburg windhoek',
```

Comment:
Why not e.g. `Africa/Lagos` here (with the upper-case letters and with the slash)?